app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(name=person["name"], age=person["age"])
+                   for person in people]
+    for person in people:
+        if person.get("husband"):
+            main_person = Person.people[person["name"]]
+            husband_person = Person.people[person["husband"]]
+            main_person.husband = husband_person
+        elif person.get("wife"):
+            main_person = Person.people[person["name"]]
+            wife_person = Person.people[person["wife"]]
+            main_person.wife = wife_person
```

Comment:
There is some duplication here. The code for assigning a husband and a wife to a person is almost identical. Consider creating a private method in the `Person` class to handle this. This would make your code DRY (Don't Repeat Yourself), easier to read, and easier to maintain.